admin-base/ui.apps/src/main/content/jcr_root/apps/field/collection/template.vue

```diff
@@ -34,15 +34,16 @@
           v-bind:class="getItemClass(item, index)"
           v-bind:key="`collection-item-${index}`"
           ref="item"
-          draggable="true"
-          v-on:dragstart="onDragStart(item, index, $event)"
-          v-on:dragover.prevent="onDragOver($event, index)"
-          v-on:dragenter.prevent="onDragEnter"
-          v-on:dragleave.prevent="onDragLeave($event, index)"
-          v-on:drop.prevent="onDrop($event, index, item)"> {{ item._opDelete }}
+          draggable="false"> {{ item._opDelete }}
         <div
             ref="header"
             class="collapsible-header"
+            draggable="true"
+            v-on:dragstart="onDragStart(item, index, $event)"
+            v-on:dragover.prevent="onDragOver($event, index)"
+            v-on:dragenter.prevent="onDragEnter"
+            v-on:dragleave.prevent="onDragLeave($event, index)"
+            v-on:drop.prevent="onDrop($event, index, item)"
```

Comment:
Don't you prefer `@drop` notation?

Reply:
@reggie7 I do indeed but I'd have to refactor all the attributes in the template then